packages/eslint-plugin-expo/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "eslint-plugin-expo",
+  "version": "0.0.1",
+  "description": "Eslint rules for Expo apps",
+  "keywords": [
+    "eslint",
+    "eslintplugin",
+    "eslint-plugin"
+  ],
+  "author": "Expo",
+  "main": "./build/index.js",
+  "files": [
+    "README.md",
+    "build"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/expo/expo.git",
+    "directory": "packages/eslint-plugin-expo"
+  },
+  "scripts": {
+    "lint": "eslint .",
+    "test": "jest",
+    "build": "tsc"
+  },
+  "dependencies": {
+    "@typescript-eslint/types": "^7.2.0",
+    "@typescript-eslint/utils": "^7.2.0",
+    "eslint": "^8.57.0"
```

Comment:
We should not need to list ESLint as a package dependency. I would recommend using `peerDependencies` instead, like we do do in `universe` config:
* https://github.com/expo/expo/blob/8523cbe1ce6562ae3ee98e6419fb5be2f10a12df/packages/eslint-config-universe/package.json#L53C1-L53C24

Reply:
I had it in peer dependencies already. I thought I needed to install it as a dev dependency as well, but `yarn -D add eslint` actually adds it to dependencies (since it's in the dependencies of the [root package.json](https://github.com/expo/expo/blob/main/package.json#L28)). Anyway looks like I can just omit it because monorpo.